integration-test/helpers/pages.js

```diff
@@ -233,10 +233,16 @@ export function loginPage (page, server, opts = {}) {
             await page.type('#email', data.username)
             await page.click('#login button[type="submit"]')
         },
-        async shouldNotPromptToSave () {
-            const calls = await page.evaluate('window.__playwright.mocks.calls')
-            // todo(Shane): is it too apple specific?
-            const mockCalls = calls.filter(([name]) => name === 'pmHandlerStoreData')
+        async shouldNotPromptToSave (platform = 'ios') {
+            let mockCalls = []
+            if (['ios', 'macos'].includes(platform)) {
+                // todo(Shane): is it too apple specific?
+                mockCalls = await mockedCalls(page, ['pmHandlerStoreData'])
+            }
+            if (platform === 'android') {
+                mockCalls = await mockedCalls(page, ['storeFormData'])
+            }
```

Comment:
I've added the Android case here and I'm using that `mockedCalls` util which seems to be doing the same thing as `evalute` then `filter`. Let me know if I missed anything, otherwise there are many other places where we could use the `mockedCalls` abstraction. I think that `todo` can also be removed?